tests/integration/list_resources_tests.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
```

Comment:
Just a note about test file names: the way pytest "discovers" tests is based on filenames -- they are to be of the form `test_*py` or end with `*_test.py`.  Probably not too important, but it would be good to stick with a convention. 

https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery

I started adding integration tests with `test_hello_fabric.py`.  Not that it is the best convention, but... would be good to stick to one convention. :-)

Reply:
Renamed the files. thank you!